components/developer/clang-38/Makefile

```diff
@@ -0,0 +1,79 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Aurelien Larcher.  All rights reserved.
+#
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME= clang-38
+LLVM_NAME=llvm
+CLANG_NAME=cfe
+LIBCXX_NAME=libcxx
+
+COMPONENT_VERSION= 3.8.1
+COMPONENT_MAJOR_VERSION= \
+  $(shell echo $(COMPONENT_VERSION) | $(GSED) -e 's/\([0-9]\+\.[0-9]\+\).*/\1/')
+COMPONENT_PROJECT_URL = http://llvm.org/
+COMPONENT_FMRI= developer/clang-3.8
```

Comment:
Perhaps, name it developer/clang-38 to be consistent and also rename clang-3.4 to clang-38 or mark obsolete?